lib/src/linden/xcolors.dart

```diff
@@ -160,4 +160,40 @@ class XColors {
       brightness == Brightness.dark ? dark : null;
 
   Brightness get brightness => _brightness;
+
+  /// right now we are in the middle of the design migration.
+  /// So far I added here some items, which I need for my tasks.
+  /// But later we will restructure all this file with proper items (names and structure)
+  //region NEW COLORS
```

Comment:
💅  You could extend this class i.e. `NewColors extends XColors` and then pass a boolean to `Linden(newColors:true)` that would switch between the implementations, so you don't need to duplicate code.

Reply:
It is not just about switching between old and new. 
Right now UI team do not have all the color mapping scheme between the old and new palette. So its more like they add something new on top.

Reply:
But yes, I can try to do this as well 😉